deeppavlov/core/common/registry.json

```diff
@@ -95,6 +96,7 @@
   "nemo_tts": "deeppavlov.models.nemo.tts:NeMoTTS",
   "ner": "deeppavlov.models.ner.network:NerNetwork",
   "ner_bio_converter": "deeppavlov.models.ner.bio:BIOMarkupRestorer",
+  "ner_chunker": "deeppavlov.models.kbqa.entity_linking:NerChunker",
```

Comment:
What is the difference between `ner-chunker` and `chunk_finder` in `deeppavlov/metrics/fmeasure.py`?

Reply:
ner_chunker splits a batch of documents into a batch of chunk lists (if the document is longer than the maximum sequence length for BERT, it is split into pieces).